### ISSUE TITLE
Portal2.fgd missing boolean defaults

### ISSUE THREAD

**Issue by Rooke6** [BEEmod/BEE2-items#4096]
### BEE2 Application Version

BEE2.4.41.0 64bit

### BEE2 Package Version

BEE2.4.41.0

### What operating system are you running on?

Windows 10 (64bit)

### Description of the bug

1. Uninstalled BEE2.4.36.0/1 from Portal 2.
2. Verified the Local Portal 2 files - successfully ( 2 files downloaded )
3. Deleted all BEE2.4.36.0/1 files from its repository folder. 
4. Ran Hammer.  It did not get any errors.  Closed Hammer.
5. Unpacked the BEE2.4.41.0 : program files, items and music packages to their repository folder.
6. Ran the BEE2.exe then exported BEE2 to Portal 2.
7. Ran Hammer and got the following errors:

File E:\Steam\steamapps\common\Portal 2\bin\portal2.fgd, line 2301: boolean type specified with nonsensical default value:
File E:\Steam\steamapps\common\Portal 2\bin\portal2.fgd, line 2474: boolean type specified with nonsensical default value:

Used Notepad++ to look at the Portal 2.fgd
The first error was in the code block for:  @SolidClass = comp_trigger_coop
And the line above  the stated line# was missing a default:
    nodamageforces(boolean) : "No damage forces?"
The 2nd error was in the code block for:  @SolidClass = comp_trigger_p2_goo
And once again the line above  the stated line# was missing a default:
    nodamageforces(boolean) : "No damage forces?"
    
Updated the fgd to include the defaults of 0 (zero) for both of those entries, and restarted Hammer, but the errors shifted down several lines in the fgd. Looking through the rest of the fgd showed lots of booleans missing defaults.

### Expected behavior

I expect Hammer to not get any errors when BEE2.4.x is installed and exported to Portal2.

### Steps to reproduce

Do everything stated in the description. 

### Logs and Reports

```shell
File E:\Steam\steamapps\common\Portal 2\bin\portal2.fgd, line 2301: boolean type specified with nonsensical default value:
File E:\Steam\steamapps\common\Portal 2\bin\portal2.fgd, line 2474: boolean type specified with nonsensical default value:
I
```

### Additional information

I'm using the Overgrown style.

**Comment by LautaroL20** [comment]
> Rooke6 said: "I expect Hammer to not get any errors when BEE2.4.x is installed and exported to Portal2."

BEEmod was not designed to export puzzles to Hammer, if you do it is in your own risk because you may experience weird issues since BEEmod is not designed to be compatible with Hammer.

**Comment by Adrthegamedev** [comment]
> LautaroL20 said: BEEmod was not designed to export puzzles to Hammer, if you do it is in your own risk because you may experience weird issues since BEEmod is not designed to be compatible with Hammer.

This isn't about exporting to hammer. Rather, this is about having hammer and beemod installed at the same time and expecting it to work like usual.

**Comment by Adrthegamedev** [comment]
@LautaroL20 I think you've misunderstood the issue here.

**Comment by Luke18033** [comment]
Yeah this issue happens just by having BEE installed and trying to use Hammer. Also, exporting BEE maps to Hammer is still a supported feature, even if it's one that we wouldn't generally recommend.
